modules/builds-basic-edit-buildconfig.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+// * builds/basic-build-operations.adoc
+
+[id="builds-basic-edit-buildconfig_{context}"]
+= Editing a BuildConfig
+
```

Comment:
What is a `BuildConfig` and why might I want to edit one?

Reply:
I'll expand on this comment. The content assumes that the reader is familiar with the "BuildConfig" term and its purpose. It almost sounds like jargon.  It would be good to start with an abstract. Maybe something like:

"A your <something> has a default build configuration, also known as a _BuildConfig_. To <give a reason>, you can edit the build configuration."